demo/index.html

```diff
@@ -12,45 +12,32 @@
 <html>
 <head>
   <title>iron-iconset-svg</title>
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
 
-  <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-  <script type="module" src="../../iron-icon/iron-icon.js"></script>
-  <script type="module" src="../iron-iconset-svg.js"></script>
-  <script type="module" src="./svg-sample-icons.js"></script>
-
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<custom-style>
-    <style is="custom-style" include="demo-pages-shared-styles"></style>
-  </custom-style>`;
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
+  <script type="module">
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+    import '@polymer/iron-icon/iron-icon.js';
+    import '../iron-iconset-svg.js';
+    import './svg-sample-icons.js';
 
-document.body.appendChild($_documentContainer.content);
-</script>
+    const $_documentContainer = document.createElement('template');
```

Comment:
we're using the `html` function for this now:
```
import {html} from '@polymer/polymer/lib/utils/html-tag.js';
const template = html``;
document.body.appendChild(template.content);
```

Reply:
Fixed.